x86_symbolic/src/Data/Macaw/X86/Crucible.hs

```diff
@@ -795,7 +796,7 @@ sDivRem sym repsz macawNum1 macawNum2 macawDenom =
     -- Get numerator
     numExt <- getNumerator dw sym macawNum1 macawNum2
     -- Get denominator
-    den <- getDenominator dw sym macawDenom
+    den <- getDenominator dw sym "idiv" macawDenom
```

Comment:
Similarly, I think we should mention that this function (`sDivRem`) is specifically meant to power the `idiv` instruction.